Source/API/CBLReplication.m

```diff
@@ -390,19 +390,42 @@ - (void) updateStatus: (CBLReplicationStatus)status
 
 #pragma mark - PUSH REPLICATION ONLY:
 
-
 - (NSSet*) pendingDocumentIDs {
-    if (!_pendingDocIDs && _started && !_pull) {
-        _pendingDocIDs = [self tellReplicatorAndWait: ^NSSet*(id<CBL_Replicator> bgReplicator) {
-            if ([_bg_replicator respondsToSelector: @selector(pendingDocIDs)])
-                return _bg_replicator.pendingDocIDs;
```

Comment:
Looks like the property `CBL_Replicator.pendingDocIDs` and its implementations can be deleted, since you've deleted the only call site.